openaev-api/src/main/java/io/openaev/rest/executor/ExecutorApi.java

```diff
@@ -59,6 +64,7 @@ public class ExecutorApi extends RestBehavior {
   private TokenRepository tokenRepository;
 
   @Resource protected ObjectMapper mapper;
+  @Autowired private ExecutorService executorService;
```

Comment:
Could we use the fact that we are modifying this class to remove all @ Autowired fields and methods and replace them by a constuctor (@ RequiredArgsConstructor) + private final fields to fit the newer coding style ?